pkg/rclone/rcserver/rc.go

```diff
@@ -600,6 +633,38 @@ func rcDeletePaths(ctx context.Context, in rc.Params) (out rc.Params, err error)
 	return out, multierr.Combine(err, statsDeleteErr)
 }
 
+// rcTransfers sets the default amount of transfers.
+// This change is not persisted after server restart.
+// Transfers correspond to the number of file transfers to run in parallel.
+// If the transfers parameter is not supplied then the transfers are queried.
+func rcTransfers(_ context.Context, in rc.Params) (out rc.Params, err error) {
+	if in["transfers"] != nil {
+		transfers, err := in.GetInt64("transfers")
```

Comment:
It looks that it will silently return current transfers value if someone call `POST core/transfers` with empty payload.
It should return error instead.
Is there any upfront payload validation ?

Reply:
This is intentional. The "built-in" endpoint responsible for controlling bandwidth works in the same way.

Reply:
Also, it always returns the transfers value.

Reply:
ok, so leave it as is, thanks !